src/ethereum/truffle/steps.js

```diff
@@ -154,65 +158,88 @@ export function writeSol (sol, fileName, requestContractsDir, fs) {
-    deployer.deploy(WitnetRequestBoard, [accounts[0]]).then(function() {
-      return deployer.deploy(WitnetRequestBoardProxy, WitnetRequestBoard.address)
-    })
+    console.log("Info: cannot deploy price feeds if no Witnet addresses are provided.")
+    process.exit(0)
   }
 }
 `;
-  fs.writeFileSync(`${migrationsDir}2_witnet_core.js`, stage2);
+  fs.writeFileSync(`${migrationsDir}/scripts/2_witnet_core.js`, stage2);
 
-  const userContractsArgs = readMigrationArgs(migrationsDir, fs);
+  const stage3 = `// WARNING: DO NOT DELETE THIS FILE
+// This file was auto-generated by the Witnet compiler, any manual changes will be overwritten.
 
-  const stage3 = `// This file was auto-generated by the Witnet compiler, any manual changes will be overwritten except
-// each contracts' constructor arguments (you can freely edit those and the compiler will respect them).
+const ERC2362PriceFeed = artifacts.require("ERC2362PriceFeed")
 const Witnet = artifacts.require("Witnet")
-const WitnetRequestBoardProxy = artifacts.require("WitnetRequestBoardProxy")
-${artifacts.map(artifact => `const ${artifact} = artifacts.require("${artifact}")`).join("\n")}
+const WitnetRequestBoard = artifacts.require("WitnetRequestBoardInterface")
+
+const radons = require("../radons")
 
 module.exports = async function (deployer) {
-  await deployer.link(Witnet, [${artifacts.join(", ")}])
-${artifacts.map(artifact => {
-    if (artifact in userContractsArgs) {
-      const args = userContractsArgs[artifact]
-      .split(/[(,)]/).slice(2).reverse().slice(1).reverse().map(x => x.trim()).join(", ");
-      console.log(`> ${artifact}: reusing existing constructor arguments (${args})`);
-      return userContractsArgs[artifact]
-    } else {
-      const args = [artifact, ...mockSolidityArgs(readSolidityArgs(artifact, userContractsDir, fs), artifacts)];
-      console.log(`> ${artifact}: generating default constructor arguments (${args.slice(1).join(", ")})
-  \x1b[33mWARNING: the autogenerated argument values may not make sense for the logic of the ${artifact}` +
-        " contract's constructor.\n  Please make sure you customize them if needed before actually deploying anything" +
-        ".\x1b[0m");
-      return `  await deployer.deploy(${args.join(", ")})`
+  await deployer.link(Witnet, [ERC2362PriceFeed])
+    const pricefeeds = Object.keys(radons)
+    for (let i = 0; i < pricefeeds.length; i ++) {
+      const pricefeed = pricefeeds[i]
+      console.log("\n> Deploying '" + pricefeed + "'...")
+      const pfc = await ERC2362PriceFeed.new(
+        WitnetRequestBoard.address,
+        radons[pricefeed].ERC2362ID
+      )
+      await pfc.setRadonScript(radons[pricefeed].bytecode)
+      console.log("  >> Contract address:\t", pfc.address)
+      console.log("  >> WRB interface:\t", WitnetRequestBoard.address)
+      console.log("  >> RADON address:\t", await pfc.radonScript.call())
+      console.log("  >> ERC2362ID:\t", radons[pricefeed].ERC2362ID)   
```

Comment:
This iterator index is serving no purpose. Let's rather use the `for...of` syntax:
```suggestion
    for (let [name, pricefeed] of Object.entries(rad)) {
      console.log("\n> Deploying '" + name + "'...")
      const pfc = await ERC2362PriceFeed.new(
        WitnetRequestBoard.address,
        pricefeed.ERC2362ID
      )
      await pfc.setRadonScript(pricefeed.bytecode)
      console.log("  >> Contract address:\t", pfc.address)
      console.log("  >> WRB interface:\t", WitnetRequestBoard.address)
      console.log("  >> RADON address:\t", await pfc.radonScript.call())
      console.log("  >> ERC2362ID:\t", pricefeed.ERC2362ID)   
```